Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/Clusterer.h

```diff
@@ -0,0 +1,54 @@
+/// \file Clusterer.h
+/// \brief Definition of the ITS cluster finder
+#ifndef ALICEO2_ITS_CLUSTERER_H
+#define ALICEO2_ITS_CLUSTERER_H
+
+#include <utility>
+#include <vector>
+
+#include "Rtypes.h"
+
+class TClonesArray;
+
+namespace o2
+{
+namespace ITS
+{
+  class PixelReader;
+  class Clusterer
+{
+ public:
+  Clusterer();
+  ~Clusterer() = default;
+
+  Clusterer(const Clusterer&) = delete;
+  Clusterer& operator=(const Clusterer&) = delete;
+
+  static void setPixelGeometry(Float_t px, Float_t pz, Float_t x0, Float_t z0) {
+    mPitchX=px; mPitchZ=pz; mX0=x0; mZ0=z0;
+  }
+  void process(PixelReader &r, TClonesArray &clusters);
+
+ private:
+  enum {kMaxRow=650}; //Anything larger than the real number of rows (512 for ALPIDE)
+  void initChip(UShort_t chipID, UShort_t row, UShort_t col, Int_t label);
+  void updateChip(UShort_t chipID, UShort_t row, UShort_t col, Int_t label);
```

Comment:
Hi again Ruben,

With the current ALPIDE response implementation, we generate a whole cluster per a MC hit.
So, every cluster (together with all digits inside it) can have only one, single, MC label.

We will consider this "labelling scheme", once we switch to the FairLinks as 
we recently discussed at GSI.   But that will be another, separate, pull request.

Cheers,
Yura.